packages/dotnet/ReadMe/HarJsonTranslationLogics/MaskHelper.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Security.Cryptography;
+using System.Text;
+
+public static class MaskHelper
+{
+    public static string Mask(string data)
+    {
+        using (SHA512 sha512 = SHA512.Create())
+        {
+            byte[] hashBytes = sha512.ComputeHash(Encoding.UTF8.GetBytes(data));
+            string base64Hash = Convert.ToBase64String(hashBytes);
+            string opts = data.Length >= 4 ? data.Substring(data.Length - 4) : data;
+            return $"sha512-{base64Hash}?{opts}";
+        }
+    }
+}
```

Comment:
@AndriiAndreiev looks like you're missing a couple end-of-file new lines. May want to update your IDE config!